include/nvexec/multi_gpu_context.cuh

```diff
@@ -66,7 +66,7 @@ namespace nvexec {
             if (op.status_ == cudaSuccess) {
               continuation_kernel<<<1, 1, 0, op.stream_>>>(std::move(op.rec_), stdexec::set_value);
             } else {
-              continuation_kernel<cudaError_t><<<1, 1, 0, op.stream_>>>(
+              continuation_kernel<<<1, 1, 0, op.stream_>>>(
```

Comment:
Some of the refactors of `continuation_kernel` which reordered the template parameters seem to have missed this one. The first template parameter is `Receiver`.

Reply:
I'm pulling this change out into its own commit, since it's unrelated.